src/common/components/contactInfo/contactInfo.tpl.html

```diff
@@ -118,11 +119,23 @@ <h4 class="panel-title  border-bottom-small  visible" translate>{{'YOUR_INFORMAT
             </label>
           </div>
         </div>
-
+        <div ng-if="$ctrl.donorDetails['donor-type'] === 'Household' && $ctrl.useV3">
+          <div class="col-sm-12 col-md-2">
+            <div class="form-group">
+              <label class="mb0 hidden-xs hidden-sm">
+                <span class="invisible mb0"></span>
+              </label>
+              <button type="button" class="btn btn-secondary btn-block" ng-click="$ctrl.toggleSpouseDetails()" uib-tooltip="{{'SPOUSE_DETAILS_TOOLTIP' | translate}}">
+                  <span ng-if="!$ctrl.showSpouseDetails">Add Spouse</span>
+                  <span ng-if="$ctrl.showSpouseDetails">Remove</span>
+              </button>
```

Comment:
Should the remove button go next to the spouse's name instead of the primary user?

Reply:
Semi-related, could we add an aria-label of "Remove Spouse" to this button so that it's clearer to users who use accessibility software what exactly is being removed? We should make sure that that label is translated as well.